packages/messenger-widget/src/components/Preferences/Notification/DeleteIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import './Notification.css';
+import deleteIcon from '../../../assets/images/delete.svg';
+import { VerificationMethod } from './bl';
+
+interface IDeleteIcon {
+    data: string;
+    type: VerificationMethod;
+    deleteNotification: Function;
```

Comment:
As long as there is no further specification on how to Notifications are implemented using the callback function to mock the behaviour is fine.
However once we have built the actual implementation that should be moved towards a hook that is handling the logic behind Notifications.